vendor/bat-native-ledger/src/bat/ledger/internal/media/twitch.cc

```diff
@@ -400,99 +390,82 @@ void MediaTwitch::OnMediaPublisherInfo(
     ledger::TwitchEventInfo new_event(twitch_info);
     new_event.status_ = GetTwitchStatus(old_event, new_event);
 
-    uint64_t real_duration = GetTwitchDuration(old_event, new_event);
+    const uint64_t real_duration = GetTwitchDuration(old_event, new_event);
     twitch_events[media_key] = new_event;
 
-    if (real_duration == 0) {
-      return;
-    }
+    SavePublisherInfo(real_duration,
+                      "",
+                      publisher_info->url,
+                      publisher_info->name,
+                      visit_data,
+                      window_id,
+                      publisher_info->favicon_url,
+                      "",
+                      publisher_info->id);
+    return;
+  }
 
-    ledger::VisitData updated_visit_data(visit_data);
-    updated_visit_data.favicon_url = "";
-    updated_visit_data.provider = TWITCH_MEDIA_TYPE;
+  if (media_id.empty()) {
+    return;
+  }
 
-    if (media_id.find("_vod_") != std::string::npos) {
-      // VOD
-      std::vector<std::string> media_props =
-          braveledger_bat_helper::split(media_id, MEDIA_DELIMITER);
-      if (media_props.empty()) {
-        return;
-      }
+  ledger::TwitchEventInfo old_event;
+  std::map<std::string, ledger::TwitchEventInfo>::const_iterator iter =
```

Comment:
Nit: Could use `auto` here to make this declaration more concise.

Reply:
done